### PR TITLE
feat(readme): removed codecov cli

### PR DESCRIPTION
**Pull request by osherv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ This package provides OpenTelemetry-compliant tracing to Javascript applications
 To install Cisco OpenTelemetry Distribution simply run:
 
 ```sh
-npm install cisco-telescope
+npm install @cisco-telescope/cisco-sdk-node
```
Comment (nemoshlag) [inline README.md:43]: did we change the decision to have the same cisco-telescope for all sdk's?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+The Conventional Commits specification is a lightweight convention on top of commit messages. It provides an easy set of rules for creating an explicit commit history; which makes it easier to write automated tools on top of. This convention dovetails with SemVer, by describing the features, fixes, and breaking changes made in commit messages. You can see examples [here](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#examples).
+We use [commitlint](https://github.com/conventional-changelog/commitlint) and [husky](https://github.com/typicode/husky) to prevent bad commit message.
+For example, you want to submit the following commit message `git commit -s -am "my bad commit"`.
+You will receive the following error :
+
+```text
+✖   type must be one of [ci, feat, fix, docs, style, refactor, perf, test, revert, chore] [type-enum]
+```
+
+Here an example that will pass the verification: `git commit -s -am "chore(cisco-telescope): update deps"`
```
Comment (samimusallam) [inline CONTRIBUTING.md:56]: there's a typo, should be "Here's an example"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+This will install the main package deps and then all the scope packages deps (and also will build the sub packages)
+
+To run all tests:
+
+```bash
+lerna run test
+```
+
+After set up the repo you will be able to work with every scope package individually (using the familiar npm commands)
```
Comment (samimusallam) [inline CONTRIBUTING.md:43]: there's a typo, should be "After setting up"

File: packages/auto-instrumentations-node/src/instrumentations/index.ts [hunk 4]
```diff
@@ -48,11 +48,6 @@ export function getCiscoNodeAutoInstrumentations(
   // TODO: once we have package per instrumentation, we will have to remove this
   //       because we will want to give the user option to config every instrumentation by itself.
   const options = _configDefaultOptions(userOptions);
-
-  if (options.debug) {
-    diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.DEBUG);
-  }
```
Comment (nemoshlag) [inline packages/auto-instrumentations-node/src/instrumentations/index.ts:54]: no logs & debug in this package?

File: CONTRIBUTING.md (hunk 2 above)
Comment (osherv) [reply CONTRIBUTING.md:56]: Fixed ty!

File: README.md (hunk 1 above)
Comment (osherv) [reply README.md:43]: Hmm I think this format is going to adjusted to all repos soon

File: packages/auto-instrumentations-node/src/instrumentations/index.ts (hunk 4 above)
Comment (osherv) [reply packages/auto-instrumentations-node/src/instrumentations/index.ts:54]: Yes, this is for OTel native users, they can set up they're own logger ;)

File: CONTRIBUTING.md (hunk 3 above)
Comment (osherv) [reply CONTRIBUTING.md:43]: Fixed ty!

**Review by nemoshlag — APPROVED** [review]

**Pull request merged**